### PR TITLE
Adding a json file for the yml file in workflow-templates

### LINKED ISSUE #9: Add a json file for the yml file in workflow-templates
- [ ] <b>Description</b>
    Add a json to describe the Github actions available as templates in the workflow-templates folder. <br>
- [ ] <b>Use cases</b>
    1. Describe the Github actions when setting it up for a new project.
    <br>
- [ ] <b>Proposed design</b>
    A json file inside the workflow-templates folder
- [ ] <b>Workaround</b>
    Manually add the workflow to each repository.
- [ ] <b>Additional information</b>
    N/A.

### PR DESCRIPTION
**Pull request by aikido-sensei**
Fixes #9

Added a json file to describe the Github actions in workflow-templates.

## Author's Checklist
- [x] **These changes don't break existing dependants.** If they do, and you suggest to proceed, 
please explain why.

## Reviewer's Checklist
- [ ] **All classes have documentation comments.** - NA
- [ ] **All methods have documentation comments.** - NA
- [ ] **No parameters are hardcoded.** Should be loaded as properties instead. If there needs to be hardcoded properties, explain why. - NA
- [ ] **All information to be logged/displayed to the user are internationalized.** If not, explain why.  - NA
- [ ] **README.md has been updated if needed.** - NA
- [ ] **setup.py version is set to the latest date** If not, explain why. - NA

## Assignee's Checklist
- [x] **All GitHub action checks have passed.** - N/A
- [x] **All reviewers have approved.**
- [x] **Relevant documentation has been updated if needed.**
- [x] **setup.py version is set to the latest date** If not, explain why. - N/A

### CODE REVIEW

File: workflow-templates/python_build.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Python Build - clean test verify",
+    "description": "pyPadaiyal CI workflow template for python projects.",
+    "iconName": "python_build.svg",
+    "categories": [
+        "Python",
+	"Python POM"
```
Comment (silvs110) [inline workflow-templates/python_build.json:7]: Remove Python POM

Comment (the-c0d3br34k3r) [inline workflow-templates/python_build.json:7]:
```suggestion
```

File: workflow-templates/combined_OS.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python on Mac OS, unbuntu or windows.
+
+name: Python build
```
Comment (the-c0d3br34k3r) [inline workflow-templates/combined_OS.yml:3]:
```suggestion
name: Python build - tests flake8
```

**Review by silvs110 — CHANGES_REQUESTED** [review]

**Review by the-c0d3br34k3r — CHANGES_REQUESTED** [review]
Squash all commits please.

File: workflow-templates/python_build.json (hunk 1 above)
Comment (aikido-sensei) [reply workflow-templates/python_build.json:7]: Done!

Comment (aikido-sensei) [reply workflow-templates/python_build.json]: Fixed!

**Comment by aikido-sensei** [comment]
> Squash all commits please.

Done!

**Review by the-c0d3br34k3r — APPROVED** [review]

**Review by silvs110 — APPROVED** [review]

**Pull request merged**